subprojects/testing-jvm/src/main/java/org/gradle/api/tasks/testing/junit/JUnitOptions.java

```diff
@@ -45,6 +45,9 @@ private static void replace(Set<String> target, Set<String> source) {
         target.addAll(source);
     }
 
+    // TODO this should probably be a Property<Boolean>
+    private boolean dryRun = false;
```

Comment:
@donat should it?

Reply:
We can't have this field as a provider, since it's hard to have object factory injected in JUnitOptions. As a solution i've added dry run public API to public-api-mutable-properties.txt file